suncityla/README.md

```diff
@@ -0,0 +1,73 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+# or
+bun dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+
+This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js) - your feedback and contributions are welcome!
+
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
+
+## Prisma ORM
+
+This project uses [Prisma](https://prisma.io) as an ORM. You can find the Prisma schema in `prisma/schema.prisma`.
+
+To get started, start a local postgres server, create a database and call it SunCityLA.
+An example of the connection URL is in the `.example.file` file.
```

Comment:
Can you add further details on how to create a database (add a link) and how to run the postgres server

Reply:
Also is there a way to share the same db without having to create one for each of us?

Reply:
If it's possible I think we should all set up local dbs for development. If we use a single db for testing it could cause issues. For example if someone breaks the db, it breaks for everyone. Let's talk about it in the meeting.

If people need help setting up a local db I can try to help. I think it would be beneficial for everyone to try and set up a local postgres db

Reply:
Awesome!

Reply:
@fmtabbara it seems we have 2 readme files and 2 folders (suncityla and solar-friend). Should we merge them together? we also have solar-friend folder remaining. I guess it's better to ger rid of it now.

Reply:
@AnyaVeselova this PR will rename the folder from solar-friend to suncityla. Having two readme files is OK.